### DISCUSSION TITLE
blog/convert-variable-font-to-static-font-using-python/

### DISCUSSION THREAD

**Discussion by anasrar** [anasrar/anasrar.github.io#38] [Giscus]
---

**Comment by auduongtuan** [comment]
Is there any option to name the output file and metadata correctly?

**Reply by anasrar** [reply]
you can pass `-o output.ttf`, for example:

```bash
fonttools varLib.mutator -o InterBold.ttf ./Inter.ttf wght=700
```

when I check the metadata using:
```bash
fc-query InterBold.ttf
```

Seems every metadata got copied correctly.

If you want to edit the metadata, sure you can use `ttx` which include in `fonttools`, check this blog post for instruction https://jasonmerino.me/articles/til-edit-font-metadata-with-ttx.
